package.json

```diff
@@ -13,6 +13,7 @@
     "@types/react-dom": "^16.9.8",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "resize-observer": "^1.0.0",
```

Comment:
Do we absolutely need this?

Reply:
Oh no, looks like there is some problem with the dependencies list
react and react-dom should be in peerDependencies.